backend/src/graphql/resolvers/GdtResolver.ts

```diff
@@ -6,6 +6,7 @@ import CONFIG from '../../config'
 import { GdtEntryList } from '../models/GdtEntryList'
 import { GdtTransactionSessionIdInput } from '../inputs/GdtInputs'
 import { apiGet } from '../../apis/HttpRequest'
+import { User as tUser } from '../../typeorm/entity/User'
```

Comment:
@Mogge and I discussed that we would like the convention `dbModel`. What do you think about that? Instead of `tUser` `dbUser` to make it clear when reading?

Reply:
yes make sense, better that t from typeorm, short and clear :)